google-cloud-bigtable/lib/google/cloud/bigtable/row_range.rb

```diff
@@ -91,7 +91,11 @@ def from key, inclusive: true
           self
         end
 
+<<<<<<< HEAD
```

Comment:
This files has 3 merge conflicts, search for `<<<<<<< HEAD` and then remove the Git markup and stale content, leaving only the comment that you want.

Reply:
OK